src/NUnitFramework/tests/nunit.framework.tests.csproj

```diff
@@ -10,12 +16,13 @@
 
     <!-- Microsoft.Bcl.Build workaround https://github.com/nunit/nunit/issues/2663 -->
     <SkipValidatePackageReferences>true</SkipValidatePackageReferences>
+  </PropertyGroup>
 
-    <UseWindowsForms>true</UseWindowsForms>
-    <UseWPF>true</UseWPF>
-
-    <!-- Suppress Microsoft.NET.Sdk.WindowsDesktop warning that netcoreapp2.1 doesn't support desktop applications -->
-    <MSBuildWarningsAsMessages>$(MSBuildWarningsAsMessages);NETSDK1105</MSBuildWarningsAsMessages>
+  <!-- Decide if we can use windows forms and wpf -->
+  <PropertyGroup Condition="$(UseWindowsFormsAndWPF)">
+      <DefineConstants>$(DefineConstants);UseWindowsFormsAndWPF</DefineConstants>
+      <UseWindowsForms>true</UseWindowsForms>
+      <UseWPF>true</UseWPF>
   </PropertyGroup>
```

Comment:
If we remove the `<UseWindowsFormsAndWPF>false</UseWindowsFormsAndWPF>` line, what's your opinion on making this a single conditional property group at the top?

```xml
<Project>
  <PropertyGroup Condition="'$(OS)' == 'Windows_NT' and '$(TargetFramework)' != 'netcoreapp2.1'">
    <UseWindowsFormsAndWPF>true</UseWindowsFormsAndWPF>
    <DefineConstants>$(DefineConstants);UseWindowsFormsAndWPF</DefineConstants>
    <UseWindowsForms>true</UseWindowsForms>
    <UseWPF>true</UseWPF>
  </PropertyGroup>
  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" Condition="'$(UseWindowsFormsAndWPF)' != 'true'" />
  ...
```
```

Reply:
That's totally right. 
Should be easier to understand what we are doing here if everything is contained in one block.
Will change it.